src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetSupportsSharing/index.tsx

```diff
@@ -0,0 +1,234 @@
+import { useUploadManager } from 'components/UploadManager';
+import { useDeleteSharedLiveMedia } from 'data/queries';
+import { createSharedLiveMedia } from 'data/sideEffects/createSharedLiveMedia/index';
+import { useDeleteUploadModal } from 'data/stores/useDeleteUploadModal';
+import { useSharedLiveMedia } from 'data/stores/useSharedLiveMedia/index';
+import { modelName } from 'types/models';
+import { Video } from 'types/tracks';
+import { report } from 'utils/errors/report';
+import { Nullable } from 'utils/types';
+import { SharedMediaItem } from './SharedMediaItem';
+
+const messages = defineMessages({
+  info: {
+    defaultMessage:
+      "This widget allows you manage presentation supports. It makes possible, to upload supports you want to share with students, with options to decide which support are available and which aren't",
+    description: 'Info of the widget used for managing and sharing supports.',
+    id: 'components.DashboardVideoLiveWidgetSupportsSharing.info',
+  },
+  title: {
+    defaultMessage: 'Supports sharing',
+    description: 'Title of the widget used for managing and sharing supports.',
+    id: 'components.DashboardVideoLiveWidgetSupportsSharing.title',
+  },
+  uploadButtonLabel: {
+    defaultMessage: 'Upload a presentation support',
+    description: 'A message indicating the role of the upload button.',
+    id: 'components.DashboardVideoLiveWidgetSupportsSharing.uploadButtonLabel',
+  },
+  noUploadedDocuments: {
+    defaultMessage: 'You have no uploaded documents yet.',
+    description:
+      "A message informing the user he hasn't any document imported yet.",
+    id: 'component.DashboardVideoLiveWidgetSupportsSharing.noUploadedDocuments',
+  },
+  confirmationModalTitle: {
+    defaultMessage: 'Delete shared media',
+    description:
+      'Title of the modal displayed when an instructor wants to delete a shared media',
+    id: 'component.SharedMediaLine.confirmationModalTitle',
+  },
+  confirmationModalText: {
+    defaultMessage: 'Are you sure you want to delete file {sharedMediaName} ?',
+    description:
+      'Title of the modal displayed when an instructor wants to delete a shared media',
+    id: 'component.SharedMediaLine.confirmationModalText',
+  },
+  updateSharedLiveMediaSucces: {
+    defaultMessage: 'Shared media updated.',
+    description:
+      'Message displayed when a shared media is successfully updated.',
+    id: 'component.SharedMediaLine.updateSharedLiveMediaSucces',
+  },
+  updateSharedLiveMediaFail: {
+    defaultMessage: 'Shared media update has failed !',
+    description: 'Message displayed when shared media update has failed.',
+    id: 'component.SharedMediaLine.updateSharedLiveMediaFail',
+  },
+});
+
+interface DashboardVideoLiveWidgetSupportsSharingProps {
+  video: Video;
+}
+
+export const DashboardVideoLiveWidgetSupportsSharing = ({
```

Comment:
```suggestion
export const DashboardVideoLiveWidgetSharedLiveMedia = ({
```

Reply:
Here, the goal was to follow the implicit widget's naming rule, consisting of giving a widget the name it has on the mockup, but translated in english :  
![image](https://user-images.githubusercontent.com/83247226/169850335-15265296-ab50-4413-9460-089a0b6abbfd.png)

Reply:
Yes but we can display what we want. Here it's a technical name and it must be accurate with what it does. `support sharing` in marsha is called `shared live media` and we must use this name everywhere we are working with this resource.